### DISCUSSION TITLE
Feature request: Please add SCSI-2 50-pin Fast-Narrow single-ended connector

### DISCUSSION THREAD

**Discussion by hdeller** [ZuluSCSI/ZuluSCSI-firmware#436] [Ideas]
It would be great if the "bigger" internal ZuluSCSI adapter would have a "SCSI-2 50-pin Fast-Narrow single-ended" connector in addition to the 50-pin SCSI-2 connector. This would allow to use the device directly in "server" machines which don't have the old SCSI connector. Maybe dropping the currently available DB25F connector?

**Comment by jsynack** [comment]
I agree that it would be nice to have different connectors.

I think to have the best benefit in server machines/etc - it would need to look and feel like a regularly mounted SCSI drive at all times (physically).

For example - if you have a need for a SCA2 connector on the back in order to slide into a disk board/slot/array/etc , it would need to be mounted in that specific dimensions/manner versus off to the side like the DB25. This would allow you to mate the ZuluSCSI into any caddy or mounting brackets that are meant for the legacy drives.

Perhaps there is someplace in between where the rear connector is a 'flexible' connector (of some style) and then you have an adapter to fit what you need? (50-pin, 68-pin, SCA, etc) ?  That will probably cause a more complex BOM for sure - and the potential for the connector adapter to fail - but it may at least prevent having many different versions of the PCB (each with their own manufacturing runs / stocking / etc). 

This does get into potentially dangerous territory of LVD vs. HVD and all the other fun of the era - So there may need to be more voltage protection as well to prevent blowing things up.

Something like SCA also carries the power, ID setting, etc. So it may be a good 'simplified' connector in that sense.  🤷‍♂️ 

Any combo of the PCB connector and the adapter connector would still need to respect the dimensions of a SCSI drive to stay within the interoperability of things that expect it to (mounts, arrays, boards). 

For example - if I look at using a ZuluSCSI on an old Sun Enterprise disk board for the E4x00/E5x00/E6x00 family - it is a SCA2 connector - and there are no Berg/4-pin power connectors available since the power arrives via SCA only (by design). 

Sorry - got a little off-track due to SCSI PTSD! :hurtrealbad: 

TL;DR - Yes - more connectors would be awesome. 👍

**Reply by aperezbios** [reply]
@hdeller I've converted this to a discussion, since it's a hardware-related product request, and not particularly firmware-specific. You wrote 

@jsynack you wrote `For example - if I look at using a ZuluSCSI on an old Sun Enterprise disk board for the E4x00/E5x00/E6x00 family - it is a SCA2 connector - and there are no Berg/4-pin power connectors available since the power arrives via SCA only (by design).`

...and I'd just like to point out that the challenge with SCA-2 connectors is the lack of availability of the device-side connector. Additionally, the current ZuluSCSI hardware designs are not capable of differential signaling. They can only operate in Single Ended mode.
